typings/secp256k1/index.d.ts

```diff
@@ -0,0 +1,159 @@
+declare module "secp256k1" {
```

Comment:
Did you take this from somewhere? We need to acknowledge all 3rd party code (e.g.: by including a header comment) and make sure the licenses are compatible.

Reply:
I wrote them with my bare hands 😛